### PR TITLE
Update to MP6 and JEE10

### PR DESCRIPTION
**Pull request by gkwan-ibm**
address issue: https://github.com/OpenLiberty/guide-openshift-codeready-containers/issues/51 or https://github.com/OpenLiberty/guides-common/issues/891

### Review changes
- [x]  pom.xml
  - java 11
  - jakarta is 10, microfile-profile is 6.0, and dependencies
  - LMP 3.8.2
- [x] server.xml
  - features version
- [x] index.html
  - license and features version
- [x] java files
  - EPL 2.0
- [x] License file
   - EPL 2.0

### End-to-end test
- [x] update lgdev with `version-update-mp6` branch
- [x] do end-to-end test and review content carefully
  - clone the `version-update-mp6` branch
  - if index.html has changes, make sure the links work

### CODE REVIEW

**Review by saumyapandyaa — DISMISSED** [review]

**Comment by saumyapandyaa** [comment]
tested end-to-end on mac and RHEL with no issues

**Review by saumyapandyaa — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -56,11 +57,9 @@ The two microservices that you'll deploy are called `system` and `inventory`. Th
 
 == Additional prerequisites
 
-Before you begin, the following tools need to be installed:
+Before you begin, the following tool needs to be installed:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Before you begin, install OpenShift Local.
```

File: README.adoc [hunk 2]
```diff
@@ -56,11 +57,9 @@ The two microservices that you'll deploy are called `system` and `inventory`. Th
...
 
-* *Docker:* You need a containerization software for building containers. Kubernetes supports various container types, but you'll use Docker in this guide. For installation instructions, refer to the official https://docs.docker.com/get-docker/[Docker documentation^].
-
-* *CodeReady Containers:* You need to install CRC to run OpenShift 4 locally on your computer. For installation instructions, refer to the official https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/1.23/html/getting_started_guide/installation_gsg[CodeReady Containers documentation].   
+* *OpenShift Local:* You need to install OpenShift Local to run OpenShift 4 locally on your computer. For installation instructions, refer to the official https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.23/html/getting_started_guide/installing[OpenShift Local documentation].
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
You must install OpenShift Local to run OpenShift 4 locally on your computer. For installation instructions, refer to the official https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.23/html/getting_started_guide/installing[OpenShift Local documentation].   
```

File: README.adoc [hunk 3]
```diff
@@ -88,14 +85,14 @@ crc setup
...
-Next, run the following command to start the CodeReady Containers virtual machine and OpenShift cluster:
+Next, run the following command to start the OpenShift Local virtual machine and OpenShift cluster:
 
 [role=command]
 ```
 crc start
 ```
 
-When prompted, supply your user pull secret. The pull secret can be found on the page where you downloaded the https://cloud.redhat.com/openshift/create/local[latest release of CodeReady Containers].
+When prompted, supply your user pull secret. The pull secret can be found on the page where you downloaded the https://cloud.redhat.com/openshift/create/local[latest release of OpenShift Local].
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Supply your user pull secret at the prompt. You can find the pull secret  on the page where you downloaded the https://cloud.redhat.com/openshift/create/local[latest release of OpenShift Local].
```

File: README.adoc [hunk 4]
```diff
@@ -117,13 +114,14 @@ Log in as user:
...
-To interact with the OpenShift cluster, you need to use the `oc` commands. CRC already includes the `oc` binary. To use the `oc` commands, run the following command to see instructions on how to configure your PATH:
+To interact with the OpenShift cluster, you need to use the `oc` commands. To build containers, you need a containerization software such as podman. OpenShift Local already includes the `oc` and `podman` binary. To use the `oc` and `podman` commands, run the following command to see instructions on how to configure your PATH:
 
 [role=command]
 ```
-crc oc-env
+crc podman-env
 ```
 
 The resulting output differs based on your OS and environment, but you should get an output similar to the following:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
The resulting output differs based on your OS and environment, but the output is similar to the following example:
```

File: README.adoc [hunk 5]
```diff
@@ -648,7 +555,7 @@ crc delete
 
 == Great work! You're done!
 
-You just deployed two microservices running in Open Liberty to OpenShift using CodeReady Containers. You also learned how to use `oc` to deploy your microservices on a {kube} cluster.
+You just deployed two microservices running in Open Liberty to OpenShift cluster using OpenShift Local. You also learned how to use `oc` to deploy your microservices on a {kube} cluster.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
You just deployed two microservices running in Open Liberty to an OpenShift cluster by using OpenShift Local. You also learned how to use `oc` to deploy your microservices on a {kube} cluster.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

File: README.adoc [hunk 6]
```diff
@@ -10,21 +10,22 @@
...
 :page-related-guides: ['kubernetes-intro', 'kubernetes-microprofile-config', 'kubernetes-microprofile-health', 'istio-intro', 'cloud-openshift-operator'] 
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/prod
 :source-highlighter: prettify
-:page-seo-title: Deploying Java microservices to Red Hat OpenShift using CodeReady Containers
-:page-seo-description: A getting started tutorial with examples on how to deploy Java microservices to a Kubernetes cluster on OpenShift running locally with CodeReady Containers using OpenShift Container Registry (OCR). 
+:page-seo-title: Deploying Java microservices to Red Hat OpenShift using OpenShift Local (formerly known as CodeReady Containers)
+:page-seo-description: A getting started tutorial with examples on how to deploy Java microservices to a Kubernetes cluster on OpenShift running locally with OpenShift Local (formerly known as CodeReady Containers) using OpenShift Container Registry (OCR). 
 :guide-author: Open Liberty
-= Deploying microservices to OpenShift using CodeReady Containers
+= Deploying microservices to OpenShift cluster using OpenShift Local
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
= Deploying microservices to an OpenShift cluster using OpenShift Local
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
LGTM except for a minor edit to the display title

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Pull request merged**